### PR TITLE
additions to SRV_ServiceIdentification

### PR DESCRIPTION
**Pull request by dblodgett-usgs**

### CODE REVIEW

File: owslib/iso.py [hunk 1]
```diff
@@ -550,8 +553,19 @@ def __init__(self, md=None):
             self.operations = []
             self.operateson = []
         else:
+            self.title=None
```
Comment (tomkralidis) [inline owslib/iso.py]: We don't need to init this here, since self.title below gets set no matter what

File: owslib/iso.py [hunk 2]
```diff
@@ -550,8 +553,19 @@ def __init__(self, md=None):
...
             val=md.find(util.nspath_eval('gmd:citation/gmd:CI_Citation/gmd:title/gco:CharacterString', namespaces))
             self.title=util.testXMLValue(val)
+            
+            self.abstract=None
```
Comment (tomkralidis) [inline owslib/iso.py]: We don't need to init this here, since `self.abstract` below gets set below no matter what via `util.testXMLValue`

**Comment by tomkralidis** [comment]
@dblodgett-usgs thanks for this.  Overall comments (in addition to line comments above):

- can you squash this PR so we don't rewrite history?
- if possible, adding a test file / unit test would be great

**Comment by dblodgett-usgs** [comment]
This this should do the trick. Sorry for the big mess of a PR before.

**Comment by tomkralidis** [comment]
Perfect!  Thanks @dblodgett-usgs

**Pull request merged**